examples/bug-detectors/command-injection/custom-hooks.js

```diff
@@ -39,6 +39,6 @@ registerReplaceHook(
 				`Command Injection in spawnSync(): called with '${command}'`
 			);
 		}
-		guideTowardsEquality(command, "jaz_zer", hookId);
+		fuzzer.tracer.guideTowardsEquality(command, "jaz_zer", hookId);
```

Comment:
For fuzzing's sake it shouldnt :laughing: On a serious note, I actually want to add a configuration option to enable this for our (OS-) command-injection bug detector, because calling `exec`, even on a dummy command that does nothing, gives us 5 executions per second, whereas not calling exec more than 20k execs/s.
The purpose of this hook was to quickly find a known command injection in the library in a matter of a few seconds, and to have a 0 chance of the fuzzer doing a "rm -rf" in my local test.
I added comments explaining why the hook does not call the original function.